tests/FakeItEasy.Specs/NonGenericCreationOptionsSpecs.cs

```diff
@@ -0,0 +1,845 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using System.Reflection.Emit;
+    using FakeItEasy.Creation;
+    using FakeItEasy.Tests;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+
+    public static class NonGenericCreationOptionsSpecs

```

Comment:
This tempts me to suggest parameterized specs, or a common base, but I wonder if it's worth it.